sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -799,8 +801,7 @@ public void statusUpdate(SchedulerDriver driver, Protos.TaskStatus status) {
                     // Map the TaskStatus to a TaskInfo. The map will throw a StateStoreException if no such
                     // TaskInfo exists.
                     try {
-                        Protos.TaskInfo taskInfo = StateStoreUtils.getTaskInfo(stateStore, status);
-                        StateStoreUtils.storeTaskStatusAsProperty(stateStore, taskInfo.getName(), status);
+                        StateStoreUtils.storeTaskStatusAsProperty(stateStore, taskName, status);
                     } catch (StateStoreException e) {
                         LOGGER.warn("Unable to store network info for status update: " + status, e);
```

Comment:
Would it make sense to log `taskName` here as well or does the output of `status` provide sufficient context?